bindings/python/scripts/generate.py

```diff
@@ -1,6 +1,8 @@
+from typing import Any, List, Dict
+import os
```

Comment:
I have separated `os` related functions in `utils.py`, so we can either define additional functions in that or shift usage here and remove from utils. I am fine either way.

Reply:
I'm working on an interface.py which will remove all the interface options there.